run-snippet-tests.sh

```diff
@@ -79,16 +79,16 @@ echo "Detected SDK: $LANGUAGE version $SDK_VERSION"
 
 case $LANGUAGE in
   js)
-    docker run -it --name runner_js --rm -e DEV_MODE="$DEV_MODE" --network ci_default --link kuzzle -v "$(pwd)":/app kuzzleio/documentation-v2:js $TESTS_PATH
+    docker run -it --name runner_js --rm -e DEV_MODE="$DEV_MODE" -e SDK_VERSION="$SDK_VERSION" -e LANGUAGE="$LANGUAGE" --network ci_default --link kuzzle -v "$(pwd)":/app kuzzleio/documentation-v2:js $TESTS_PATH
```

Comment:
Why did you have to specify the SDK_VERSION and the LANGUAGE ? Both can be deducted from TESTS_PATH (and LANGUAGE from the container image itself)

Reply:
It's easier to pass them to container as they're already calculated in this script